.stylelintrc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": [
+      true,
+      {
+        "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
+      }
+    ],
+    "scss/at-rule-no-unknown":  [
+      true,
+      {
+        "ignoreAtRules": ["tailwind", "apply", "variants", "responsive", "screen"]
+      }
+    ],
+    "csstree/validator": true,
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true
```

Comment:
- [ ] Please do not modify linter configuration files in any way.

Reply:
I have tried every way to solve it. Not working 
please guide